vmdb/app/models/miq_provision_virt_workflow.rb

```diff
@@ -1385,6 +1367,19 @@ def self.from_ws_ver_1_x(version, userid, template_fields, vm_fields, requester,
     $log.error "#{log_header}: <#{err}>"
     raise err
   end
+
+  private
+
+  def exit_pre_dialog
+    @running_pre_dialog              = false
+    @values[:pre_dialog_vm_tags]     = @values[:vm_tags].dup
+    @values[:forced_sysprep_enabled] = 'fields' if @values[:sysprep_enabled] == 'fields'
```

Comment:
Oh wow, thanks.  I misread that as a @values nil check that wasn't happening anymore.